subcommands/targets/prune.go

```diff
@@ -26,6 +26,21 @@ func init() {
 		Short: "Prune target(s)",
 		Run:   doPrune,
 		Args:  cobra.MinimumNArgs(1),
+		Example: `
+  # prune a single target name its name:
```

Comment:
```suggestion
  # prune a single target by its name:
```

Reply:
i think other parts of fioctl talk about `target name` and `target version`. So maybe we do `prune a single target name`

Reply:
This is what happens when one spends less then 5 minutes on the commit.
Fixed in force-push.